scripts/wrappers/start-exporter.sh

```diff
@@ -9,6 +9,12 @@ function fetch_exporter_args () {
     DASHBOARDS_PORT=$( get_yaml_prop "${OPENSEARCH_DASHBOARDS_PATH_CONF}/opensearch_dashboards.yml" "server.port" "5601" )
     DASHBOARDS_USERNAME=$( get_yaml_prop "${OPENSEARCH_DASHBOARDS_PATH_CONF}/opensearch_dashboards.yml" "opensearch.username" "kibanaserver" )
     DASHBOARDS_PASSWORD=$( get_yaml_prop "${OPENSEARCH_DASHBOARDS_PATH_CONF}/opensearch_dashboards.yml" "opensearch.password" "kibanaserver" )
+    SCHEME=$(snapctl get scheme)
+
+    TLS_ARGS=""
+    if [ "${SCHEME}" == "https" ]; then
+        TLS_ARGS="-kibana.skip-tls true"
```

Comment:
We need to review this - in a future PR - and understand how to make self-signed certificates work with the exporter without skipping the verification. 
For ref. https://github.com/chamilad/kibana-prometheus-exporter/blob/df0ed073981a5fb4394948d663ed42300c9f8f27/exporter/collector.go#L147

Reply:
I can also open an issue and/or contact the maintainer to ask if there may be any plans on their side.